infoschema/slow_log.go

```diff
@@ -87,6 +88,7 @@ func parseSlowLogFile(tz *time.Location, filePath string) ([][]types.Datum, erro
 func ParseSlowLog(tz *time.Location, reader *bufio.Reader) ([][]types.Datum, error) {
 	var rows [][]types.Datum
 	startFlag := false
+	prevStmtPrefix := variable.SlowLogPrevStmt + variable.SlowLogSpaceMarkStr
```

Comment:
Please add `SlowLogPrevStmt ` in `setFieldValue` function. No need to change this function.